### PR TITLE
task 3 lint added commit

### PR DESCRIPTION
**Pull request by ishika123mathur**

### CODE REVIEW

**Comment by GargPermil** [comment]
lint needs to be configured as well.

**Review by ishika123mathur — COMMENTED** [review]
configuration done for lint

File: eslint.config.mjs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import globals from "globals";
```
Comment (GargPermil) [inline eslint.config.mjs:1]: why file name is mjs

**Review by ishika123mathur — COMMENTED** [review]
The .mjs extension explicitly tells Node.js that the file should be treated as an ECMAScript module. but now converted to js.please check

**Review by GargPermil — APPROVED** [review]

**Comment by GargPermil** [comment]
image

**Pull request merged**